conf/prod.routes

```diff
@@ -1,6 +1,8 @@
 # Add all the application routes to the app.routes file
 POST       /nsihts/createaccount                      uk.gov.hmrc.helptosavestub.controllers.NSIController.createAccount()
 PUT        /nsihts/updateemail                        uk.gov.hmrc.helptosavestub.controllers.NSIController.updateEmail()
+PUT        /nsihts/test                               uk.gov.hmrc.helptosavestub.controllers.NSIController.test()
```

Comment:
Pls can you change /test to /healthcheck, and NSIController.test() to NSIController.healthCheck(), thx